### ISSUE TITLE
Can't set default related entry

### ISSUE THREAD

**Issue by webrgp** [craftcms/feed-me#1170]
### Description

On the "Map" step, when trying to configure a default entry relation, I get this error in Ajax

Here is the json response:

```json
{
  "message": "Setting unknown property: craft\\elements\\db\\EntryQuery::enabledForSite",
  "exception": "yii\\base\\UnknownPropertyException",
  "file": "/var/www/html/cms/vendor/yiisoft/yii2/base/Component.php",
  "line": 209,
  "trace": [
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/elements/db/ElementQuery.php",
      "line": 523,
      "function": "__set",
      "class": "yii\\base\\Component",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/elements/db/EntryQuery.php",
      "line": 230,
      "function": "__set",
      "class": "craft\\elements\\db\\ElementQuery",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/yiisoft/yii2/BaseYii.php",
      "line": 558,
      "function": "__set",
      "class": "craft\\elements\\db\\EntryQuery",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/controllers/ElementIndexesController.php",
      "line": 566,
      "function": "configure",
      "class": "yii\\BaseYii",
      "type": "::"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/controllers/ElementIndexesController.php",
      "line": 108,
      "function": "elementQuery",
      "class": "craft\\controllers\\ElementIndexesController",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/yiisoft/yii2/base/Controller.php",
      "line": 176,
      "function": "beforeAction",
      "class": "craft\\controllers\\ElementIndexesController",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/yiisoft/yii2/base/Module.php",
      "line": 552,
      "function": "runAction",
      "class": "yii\\base\\Controller",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/web/Application.php",
      "line": 301,
      "function": "runAction",
      "class": "yii\\base\\Module",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/web/Application.php",
      "line": 625,
      "function": "runAction",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/craftcms/cms/src/web/Application.php",
      "line": 280,
      "function": "_processActionRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/vendor/yiisoft/yii2/base/Application.php",
      "line": 384,
      "function": "handleRequest",
      "class": "craft\\web\\Application",
      "type": "->"
    },
    {
      "file": "/var/www/html/cms/web/index.php",
      "line": 12,
      "function": "run",
      "class": "yii\\base\\Application",
      "type": "->"
    }
  ],
  "error": "Setting unknown property: craft\\elements\\db\\EntryQuery::enabledForSite"
}
```

### Additional info

- Craft version: 4.1.4.1
- PHP version: 8.1
- Database driver & version: MySql 8
- Plugins & versions: 
  - FeedMe 5.0.4

**Comment by jamesmacwhite** [comment]
I believe enabledForSite was deprecated in Craft CMS 3.5 and is removed in Craft CMS 4: https://docs.craftcms.com/api/v4/craft-elements-db-entryquery.html#public-properties. It no longer exists in the class reference.

**Comment by webrgp** [comment]
I am still having the same issue on a different project 🤷🏻‍♂️

**Comment by jamesmacwhite** [comment]
Yeah the Craft CMS 4 FeedMe version has references to the enabledForSite property which aren't valid anymore as the property is gone.

**Comment by webrgp** [comment]
What's the best course of action here? Can I submit a PR for it?

**Comment by jamesmacwhite** [comment]
PR to remove the enabledForSite property references and replace with status or equivalent.

Or fork the plugin and update your composer file to point to the forked repo to fix the issue temporarily until fixed upstream.

**Comment by davorpeic** [comment]
If you create a fork, share a link :)

**Comment by i-just** [comment]
This was fixed in version 5.0.5: https://github.com/craftcms/feed-me/blob/main/CHANGELOG.md#505---2023-01-09
